agent/consul/usagemetrics/usagemetrics.go

```diff
@@ -145,6 +149,7 @@ func (u *UsageMetricsReporter) Run(ctx context.Context) {
 }
 
 func (u *UsageMetricsReporter) runOnce() {
+	u.logger.Debug("Starting usage run")
```

Comment:
```suggestion
	u.logger.Trace("Starting usage run")
```

Do you think trace level would be more appropriate than debug here?

Reply:
No strong opinions